### PR TITLE
Fix running with `time_limits` and `memory_limits` present in config

### PR DESCRIPTION
**Pull request by MasloMaslane**
I kinda forgot that not all tests have to have the same time and memory limits. This PR fixes that.

### CODE REVIEW

File: src/sinol_make/commands/run/__init__.py [hunk 1]
```diff
@@ -63,9 +63,14 @@ def print_view(term_width, term_height, program_groups_scores, all_results, prin
     sys.stdout = output
 
     program_scores = collections.defaultdict(int)
-    program_times = collections.defaultdict(lambda: -1)
-    program_memory = collections.defaultdict(lambda: -1)
-    time_remaining = (len(executions) - print_data.i - 1) * 2 * time_limit / cpus / 1000.0
+    program_times = collections.defaultdict(lambda: (-1, 0))
```
Comment (tonowak) [inline src/sinol_make/commands/run/__init__.py:66]: What is `(-1, 0)`? I don't see any definitions here, so it's hard to confirm the code.

File: src/sinol_make/helpers/package_util.py [hunk 2]
```diff
@@ -49,3 +49,37 @@ def get_executable_path(solution: str) -> str:
...
+    test_group = get_group(test_path)
+
+    if "time_limits" in config:
+        if test_id in config["time_limits"]:
+            return config["time_limits"][test_id]
+        elif int(test_group) in config["time_limits"]:
+            return config["time_limits"][int(test_group)]
+        elif str(test_group) in config["time_limits"]:
+            return config["time_limits"][str(test_group)]
+    return config["time_limit"]
```
Comment (tonowak) [inline src/sinol_make/helpers/package_util.py:68]: That looks weird, is there really no better way to write this code?

File: src/sinol_make/commands/run/__init__.py (hunk 1 above)
Comment (MasloMaslane) [reply src/sinol_make/commands/run/__init__.py:70]: I added a comment describing it

File: src/sinol_make/helpers/package_util.py (hunk 2 above)
Comment (MasloMaslane) [reply src/sinol_make/helpers/package_util.py]: I changed it a bit

**Review by tonowak — APPROVED** [review]

**Pull request merged**